site/src/AppRouter.tsx

```diff
@@ -177,127 +188,145 @@ export const AppRouter: FC = () => {
 
           {/* Dashboard routes */}
           <Route element={<RequireAuth />}>
-            <Route element={<DashboardLayout />}>
-              <Route index element={<IndexPage />} />
+            <Route element={<AuthenticatedProviders />}>
```

Comment:
This is the main route change. Added a provider to all authenticated routes.

Reply:
We should not need this, `<RequireAuth>` is exactly for that.